app/forms/pages/question_form.rb

```diff
@@ -1,13 +1,24 @@
 class Pages::QuestionForm < BaseForm
   include QuestionTextValidation
 
-  attr_accessor :question_text, :hint_text, :is_optional, :answer_type
+  attr_accessor :question_text, :hint_text, :is_optional, :answer_type, :draft_question
 
   # TODO: We could lose these attributes once we have an Check your answers page
   attr_accessor :answer_settings, :page_heading, :guidance_markdown
 
   # TODO: Remove once we get rid of session storage
   attr_accessor :form_id, :page_id
 
+  validates :draft_question, presence: true
   validates :hint_text, length: { maximum: 500 }
+
+  def submit
+    return false if invalid?
+
+    draft_question.question_text = question_text
+    draft_question.hint_text = hint_text
+    draft_question.is_optional = is_optional
+    draft_question.question_text = question_text
```

Comment:
Could you do this with `merge` (or some other syntax) to make this more compact and less repetitive? I'm thinking maybe something like:
```suggestion
    draft_question.merge({ question_text:, hint_text:, is_optional:, question_text: })
```

Reply:
Thanks for the suggestion, I'll make the changes.....interesting that both of us didn't catch the code setting `question_text` twice

Reply:
Oh yeah 😅

Reply:
@DavidBiddle I've pushed a commit which should make it easier to read without all the repetition